sat/cli/bmccreds/creds_manager.py

```diff
@@ -67,12 +67,15 @@ class BMCCredsManager:
     CHASSIS_BMC_XNAME_REGEX = re.compile(r'x\d+c\d+b\d+$')
     ROUTER_BMC_XNAME_REGEX = re.compile(r'x\d+c\d+r\d+b\d+$')
 
-    def __init__(self, password, xnames, domain, force, report_format):
+    def __init__(self, password, xnames, domain, force, report_format, length=RANDOM_PASSWORD_LENGTH,
+                 allowed_chars=VALID_CHAR_SETS):
```

Comment:
This default value is not the correct type. It should be a string.

Let's add the arguments to a docstring for this constructor.

Reply:
Addressed